Docker/README-multiple-nodes.md

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I would suggest to add this as a section in the existing `README` file so we minimise the file fragmentation. Really nice though adding this!

Reply:
Done.